src/Foundatio/Queues/SharedQueueOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Foundatio.Serializer;
+using Foundatio.Utility;
+using Microsoft.Extensions.Logging;
+
+namespace Foundatio.Queues {
+    public interface ISharedQueueOptions {
+        string Name { get; set; }
+        int Retries { get; set; }
+        TimeSpan WorkItemTimeout { get; set; }
+    }
+
+    public class SharedQueueOptions<T> : SharedOptions, ISharedQueueOptions where T : class {
+        public string Name { get; set; } = typeof(T).Name;
+        public int Retries { get; set; } = 2;
+        public TimeSpan WorkItemTimeout { get; set; } = TimeSpan.FromMinutes(5);
+        public IEnumerable<IQueueBehavior<T>> Behaviors { get; set; }
+    }
+
+    public interface ISharedQueueOptionsBuilder : ISharedOptionsBuilder {}
```

Comment:
I think the interface `ISharedQueueOptionsBuilder` is not neccessary. We can define the extension methods for `IOptionsBuilder` instead.